apps/hpe-csi/data.yaml

```diff
@@ -0,0 +1,39 @@
+tags:
+  - Storage
+  - Drivers and plugins
+title: "HPE CSI Driver"
+summary: "TODO"
+logo: https://raw.githubusercontent.com/hpe-storage/co-deployments/master/docs/assets/hpedev.png
+description: |
+    TODO
```

Comment:
HPE CSI Driver for Kubernetes allows users to dynamically provision persistent and ephemeral storage with support for a wide range of storage products from HPE, including Alletra Storage MP B10000 and Alletra 5000/6000/9000. Advanced data services are provided to accelerate data management, such snapshots, replication, clones, and import existing data into workloads running on Kubernetes.

Reply:
Ok, thank you